plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/command/valueproviders/CurrentProjectPathProvider.java

```diff
@@ -41,7 +42,7 @@
                                                    ProjectReadyHandler,
                                                    CurrentProjectChangedHandler {
 
-    private static final String KEY = "${current.project.path}";
+    public static final String KEY = "${current.project.path}";
```

Comment:
Because I had to place this property name in other place, such as Preview Url template.